src/viewconditions.rs

```diff
@@ -48,13 +48,7 @@ impl ViewConditions {
     /// Hyperbolic post-adaptation response compression function
     /// 
     /// As used in CIECAM02 and CAM16
-    /*
-    pub fn lum_adapt(&self, v: &mut f64) {
-        let t = (self.f_l() * *v / 100.).powf(0.42);
-        *v = v.signum() * 400. * t / (27.13 + t) + 0.1;
-    }
-     */
-
```

Comment:
I was not certain what the intention here was. This code is a regular comment, inside a doc comment. It did not look like it was intended to be a doc comment code example, so I just removed it. Was this correct?